docs/premium-features/break-the-monolith.md

```diff
@@ -6,61 +6,64 @@ slug: /break-the-monolith
 
 # Break The Monolith
 
-Amplication's new "Break The Monolith" feature helps you modernize monolithic architectures and transform them into microservices. Using AI, Amplication simplifies the transition of your entities to different services, streamlining your shift to a microservices architecture.
+Amplication's "Break The Monolith" feature helps you modernize monolithic architectures and transform them into microservices. It introduces a new drag-and-drop interface to simplify the process of moving entities across different services.
 
-This guide will teach you everything so you can best utilize this feature for your apps.
+Additionally, you have the option to utilize our new AI Helper for further guidance and automation in this transition, optimizing the process of moving to a microservices architecture.
+
+This guide will show you how to effectively use this feature for your services.
 
 :::note
-The "Break The Monolith" feature is available exclusively to Enterprise workspaces, with trial workspaces having limited access.
+The "Break The Monolith" feature is **available exclusively to Enterprise workspaces including trial workspaces**.
 :::
 
 ## What is "Break The Monolith"?
 
-"Breaking the Monolith" is designed to help you transition a legacy app with a monolithic structure to a microservice architecture.
+"Break The Monolith" helps you transition any monolithic service, including monolithic legacy systems, to a modular microservice architecture. It allows for both manual and AI-driven restructuring of services.
 
-This feature consists of two parts:
+"Breaking the Monolith" is designed to help you transition any app with a monolithic structure to a microservice architecture.
 
-1. **AI-powered Entity Analysis**: We assess your service's entities to suggest optimal division into microservices.
-2. **Service Migration**: Based on that analysis, we migrate entities across services.
+This feature focuses on the following 3 parts:
 
-:::note
-**This feature uses LLMs to analyze and suggest ideal microservices structures**. It requires sharing details of your entities and fields with LLMs for analysis. We ensure the highest privacy standards and use your data only to improve your project's architecture.
-:::
+1. **Manual Service Redesign**: Manually move entities between existing or new services to create tailored micro-services.
+2. **AI-powered Entity Analysis and Service Suggestions**: As an optional feature, AI analyzes and proposes the most optimal way to distribute your entities among services, offering smart recommendations for service composition and entity organization.
+3. **Service and Database Creation**: Based on the new architecture, Amplication creates new services and updates databases to support service separation.
```

Comment:
1. I wouldn't call it "service and database creation" - but something like "applying the new architecture" or "creating the micro-services" or such. 
2. "Amplication creates new services and updates databases to support service separation." - Amplication creates new services, with their entities, APIs, DTOs etc. to support the new microservice architecture